src/simulators/tensor_network/svd.cpp

```diff
@@ -478,48 +474,9 @@ status csvd(cmatrix_t &A, cmatrix_t &U,rvector_t &S,cmatrix_t &V)
 	      }
 	if( ! equal )
 	{
-//		temp_A.SetOutputStyle(Matrix);
-//		diag_S.SetOutputStyle(Matrix);
-		cout << "error: wrong SVD calc: A != USV*" << endl;
-//		cout << "A = " << endl;
-//		cout << temp_A;
-//		cout << "U = " << endl;
-//		cout << U;
-//		cout << "S = " << endl;
-//		cout << diag_S;
-//		cout << "V* = " << endl;
-//		cout << AER::Utils::dagger(V);
-//		cout << "USV* = " << endl;
-//		cout << U*diag_S*(AER::Utils::dagger(V));
-//		cout << temp;
-//		assert(false);
+	  cout << "error: wrong SVD calc: A != USV*" << endl;
```

Comment:
I think we need to put this into `#ifdef DEBUG` directives, right?
Is this a real error? Should we throw an exception here?... otherwise, I would change "error" with something different: "warning" ?

Reply:
This is a real error. If we get this error, the algorithm will not continue correctly., so it is better to exit. I changed to a "throw". Let me know if you think this should be handled differently.